localization_topic_name: /localization/utmpose

chassis_state_topic_name: /chassis_state

control_command_topic_name: /control_cmd

log_trigger_topic_name: /control/trigger

use_trigger: true

node_rate: 40   # [Herz]
